docs/snippets/common/test-runner-a11y-disable.ts.mdx

```diff
@@ -1,14 +1,13 @@
 ```ts
 // .storybook/test-runner.ts
 
-import { injectAxe, checkA11y } from 'axe-playwright';
-
+import type { TestRunnerConfig } from '@storybook/test-runner';
 import { getStoryContext } from '@storybook/test-runner';
 
-import type { TestRunnerConfig } from '@storybook/test-runner';
+import { injectAxe, checkA11y } from 'axe-playwright';
 
 /*
- * See https://storybook.js.org/docs/react/writing-tests/test-runner#test-hook-api-experimental
+ * See https://storybook.js.org/docs/react/writing-tests/test-runner#test-hook-api
  * to learn more about the test-runner hooks API.
  */
 const a11yConfig: TestRunnerConfig = {
```

Comment:
I'd say we should just call this config. A11y is one thing that you can do with it, but you can do multiple things as well. This way the snippets and usage are all familiar to users.